app/models/search_models.py

```diff
@@ -5,7 +5,7 @@
 @dataclass_json
 @dataclass
 class SearchQuery:
-    keystroke: str
```

Comment:
This is ambigous. We can search labels or headers while using this field. I'd change it to search_term (it's used by elastic in their frontend-ui implementation and seems specific to me)